data-reconciliation-app/test/test.sh

```diff
@@ -88,29 +88,70 @@ cd "${certificate_dir}"
+printf "\n\n"
 
 memberName="member1"
-check_eq "$memberName - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../data-samples/${memberName}_data.json" $only_status_code)"
+check_eq "$memberName - data ingest should succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../test/data-samples/${memberName}_data.json" $only_status_code)"
+curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../test/data-samples/${memberName}_data.json"
+printf "\n\n"
 
 memberName="member2"
-check_eq "$memberName - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../data-samples/${memberName}_data.json" $only_status_code)"
+check_eq "$memberName - data ingest should succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../test/data-samples/${memberName}_data.json" $only_status_code)"
+printf " Response: "
+curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../test/data-samples/${memberName}_data.json"
+printf "\n\n"
 
 memberName="member2"
-check_eq "$memberName - data ingest failed (data length is zero)" "400" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "[]" $only_status_code)"
+check_eq "$memberName - data ingest should fail (data length is zero)" "400" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "[]" $only_status_code)"
+printf " Response: "
+curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "[]"
+printf "\n\n"
 
 memberName="member2"
-check_eq "$memberName - data ingest failed (data is null or empty)" "400" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "" $only_status_code)"
+check_eq "$memberName - data ingest should fail (data is null)" "400" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "" $only_status_code)"
+printf " Response: "
+curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary ""
+printf "\n\n"
+
 
+printf "\n -------- Test Reporting Service --------  \n\n"
 
-# printf "\n\n✅ Member 1 - read data report \n"
-# curl $server/app/report -X GET $(cert_arg "member1")
+reportingUrl="$server/app/report";
+
+memberName="member0"
+check_eq "$memberName - Getting all data records should succeed" "200" "$(curl $reportingUrl -X GET $(cert_arg $memberName) -H "Content-Type: application/json" $only_status_code)"
+printf " Response: "
+curl $server/app/report -X GET $(cert_arg $memberName)
+printf "\n\n"
```

Comment:
I agree with this API being printed in the terminal, just questioned the ingestion API :)